dist/twine.js

```diff
@@ -181,12 +185,18 @@
       }
     };
     Twine.refreshImmediately = function() {
-      var element, key;
+      var callbacks, cb, element, j, key, len;
       refreshQueued = false;
       for (key in elements) {
         element = elements[key];
         refreshElement(element);
       }
+      callbacks = refreshCallbacks;
+      refreshCallbacks = [];
+      for (j = 0, len = callbacks.length; j < len; j++) {
+        cb = callbacks[j];
+        cb();
```

Comment:
So, we're never removing them from the array. Does that always make sense?

I think there are places in our code right now using `refreshImmediately` and then doing something right afterwards expecting it to be done, but they don't necessarily want the code to be run after *every* refresh.

Maybe we should view a passed callback as wanting to execute after *this* refresh, and then provide a `twine:after-refresh` event to bind to for cases where you might want to have something happen after *every* refresh?